proxy_config.c

```diff
@@ -177,6 +178,7 @@ int _start_proxy_config_threads(proxy_ctx_t *ctx) {
         pthread_mutex_unlock(&ctx->manager_lock);
```

Comment:
nit: In line 176 above, the log line should say:
"Failed to start proxy manager thread"